andela/isildur-cfh
# #150398707 Fix Base App Functionality

#### What does this PR do?
- Fix Base app for the team
- Fix Eslint issues to a bare minimum
- Fix Gulp Integration (remaining test gulp task as test still currently runs with grunt)
- Invite only Eleven people

#### Description of Task to be completed?
setup base app and implement invitation of only eleven users
#### How should this be manually tested?
- run the app locally and interact with the app
- run eslint, test script and view the outputs
- check hound reports on the repo
#### Any background context you want to provide?
This will serve as a new base app for the team to further there work on
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/150398694
https://www.pivotaltracker.com/story/show/150398695
https://www.pivotaltracker.com/story/show/150398703
https://www.pivotaltracker.com/story/show/150398707
